db/schema.rb

```diff
@@ -383,6 +390,7 @@
   add_foreign_key "case_logs", "organisations", column: "owning_organisation_id", on_delete: :cascade
   add_foreign_key "case_logs", "schemes"
   add_foreign_key "locations", "schemes"
+  add_foreign_key "schemes", "organisations", column: "managing_organisation_id"
```

Comment:
Already merged in, not sure why it's showing up here as this is now conistent with main

Reply:
Do you just need to rebase your branch?

Reply:
Of course, thanks :)